cpp/src/gandiva/precompiled/string_ops_test.cc

```diff
@@ -53,6 +53,37 @@ TEST(TestStringOps, TestAscii) {
   EXPECT_EQ(ascii_utf8("999", 3), 57);
 }
 
+TEST(TestStringOps, TestChr) {
+  // CHR
+  gandiva::ExecutionContext ctx;
+  uint64_t ctx_ptr = reinterpret_cast<gdv_int64>(&ctx);
+  int32_t out_len = 0;
+
+  auto out = chr_int32(ctx_ptr, 88, &out_len);
```

Comment:
Also add unit tests for punctuations, non printable control codes, non-ascii number (>127)

Reply:
Done, I added unit tests for this cases too

Reply:
Please add a test for 0 and 255